src/CharacterList.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CharacterList from './CharacterList';
+
+it('renders without crashing', () => {
+  shallow(<CharacterList params={{}} />);
+});
+
+it('Increments page when clicked', () => {
+  const component = shallow(
+    <CharacterList params={{}} />
+  );
+
+  expect(component.state('page')).toEqual(0);
+
+  component.setState({
```

Comment:
so, I am setting the default state because I am mimicking user actions on buttons. Those buttons don't exist unless I set state. Then I am checking that state changes. Maybe line 14 is what I don't need?